helpers/multi.ts

```diff
@@ -0,0 +1,5 @@
+export type MultiReturn<T extends any[]> = T & {
+    " __multiBrand": never;
+};
+
+export declare function multi<T extends any[]>(...values: T): MultiReturn<T>;
```

Comment:
Would it be possible to provide `.js` files with some js implementation?

Reply:
Attempted an implementation, would this be for unit testing?

Reply:
That's one use case, yes. It also should be useful for universal Lua/JS modules. `multi` is probably not that useful, but something like `forRange` might be